spec/automation/unit/method_validation/parse_provider_category_spec.rb

```diff
@@ -0,0 +1,93 @@
+describe "parse_provider_category" do
+  before(:each) do
+    @user  = FactoryGirl.create(:user_with_group)
+    @inst  = "/System/Process/parse_provider_category"
+  end
+
+  let(:infra_ems)              { FactoryGirl.create(:ems_microsoft_with_authentication) }
```

Comment:
@tinaafitz should we make the ems consistent with the vm? we have infra_vm as vm_ware and ems as microsoft